renderlib/glsl/GLBasicVolumeShader.cpp

```diff
@@ -17,7 +17,7 @@ GLBasicVolumeShader::GLBasicVolumeShader()
 {
   vshader = new GLShader(GL_VERTEX_SHADER);
   vshader->compileSourceCode(R"(
-#version 330 core
+#version 400 core

```

Comment:
the changes in this directory of this type constitute the actual bug fix.  All other changes are simple renaming/moving files.